apps/docs/content/components/Stack.mdx

```diff
@@ -150,15 +150,15 @@ E.g. Providing only `narrow` will apply that value to all larger breakpoints, bu
   }}
 >
   <img
-    src="https://via.placeholder.com/150x75/f5f5f5/f5f5f5.png"
+    src="https://via.placeholder.com/150x75/D0D7DE/D0D7DE.png"
```

Comment:
Is `D0D7DE` a position on our greyscale? Not a big deal if not, but would be nice to make it that so it's not arbitrary.

Reply:
It is or at least I got it from the colors page in our Figma file